keyboards/ckeys/handwire_101/keymaps/default/keymap.c

```diff
@@ -0,0 +1,179 @@
+#include "handwire_101.h"
+#include "action_layer.h"
+
+#define _BASE           0  // base layer
```

Comment:
For Lines 4-10:

You can use an enum here. Not required, but just a tip.

```c
enum layers {
  _BASE,
  _LAYERS,
  _MUSIC,
  _MUSIC_4_LIFE, 
  _MOUSE,
  _TERMINAL,
  _ADMIN
};
```

Inline comments are still valid inside the enum.